components/Nav/index.js

```diff
@@ -1,25 +1,34 @@
 
-import CheckboxWithLabel from '../CheckboxWithLabel';
+import { useContext } from 'react';
 import SwitchWithLabel from '../SwitchWithLabel';
+import CheckboxWithLabel from '../CheckboxWithLabel';
+import AnimationEmulationContext from '../AnimationEmulationContext';
 
 const githubLink = {
   label: 'GitHub',
   href: 'https://github.com/alexander-dev-hub/react-next-motion',
```

Comment:
Please change the URL to https://github.com/anton-karlovskiy/cna-memory-considerate-animation